services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -117,255 +98,49 @@ export const rateErrorHandling = (
-        fileItems: FileItemT[],
-        index: number
-    ): { key: string; error: string } | undefined => {
-        const hasValidFiles =
-            fileItems.length > 0 &&
-            fileItems.every((fileItem) => fileItem.status === 'UPLOAD_COMPLETE')
-        const hasLoadingFiles =
-            fileItems.some((fileItem) => fileItem.status === 'PENDING') ||
-            fileItems.some((fileItem) => fileItem.status === 'SCANNING')
-
-        const documentsErrorMessage = hasLoadingFiles
-            ? 'You must wait for all documents to finish uploading before continuing'
-            : fileItems.length === 0
-            ? 'You must upload at least one document'
-            : !hasValidFiles
-            ? 'You must remove all documents with error messages before continuing'
-            : undefined
-
-        if (documentsErrorMessage) {
-            return {
-                key: `rateInfos.${index}.rateDocuments`,
-                error: documentsErrorMessage,
-            }
-        }
-        return undefined
-    }
-
-    const initialFileItemsMatrix: FileItemT[][] = draftSubmission.rateInfos.map(
-        (rateInfo) => fileItemsFromRateInfo(rateInfoFormValues(rateInfo))
-    )
-
-    const [fileItemsMatrix, setFileItemsMatrix] = React.useState<FileItemT[][]>(
-        initialFileItemsMatrix
-    )
-
-    const allRatesHasValidFiles =
-        fileItemsMatrix.every((fileItems) => fileItems.length > 0) &&
-        fileItemsMatrix
-            .flat()
-            .every((fileItems) => fileItems.status === 'UPLOAD_COMPLETE')
-
-    const showFileUploadError = shouldValidate && !allRatesHasValidFiles
-
-    const handleDeleteFile = async (key: string) => {
-        const isSubmittedFile =
-            previousDocuments &&
-            Boolean(
-                previousDocuments.some((previousKey) => previousKey === key)
-            )
-
-        if (!isSubmittedFile) {
-            const result = await deleteFile(key, 'HEALTH_PLAN_DOCS')
-            if (isS3Error(result)) {
-                throw new Error(`Error in S3 key: ${key}`)
-            }
-        }
-    }
-
-    const handleUploadFile = async (file: File): Promise<S3FileData> => {
-        const s3Key = await uploadFile(file, 'HEALTH_PLAN_DOCS')
-
-        if (isS3Error(s3Key)) {
-            throw new Error(`Error in S3: ${file.name}`)
-        }
-
-        const s3URL = await getS3URL(s3Key, file.name, 'HEALTH_PLAN_DOCS')
-        return { key: s3Key, s3URL: s3URL }
-    }
-
-    const handleScanFile = async (key: string): Promise<void | Error> => {
-        try {
-            await scanFile(key, 'HEALTH_PLAN_DOCS')
-        } catch (e) {
-            if (isS3Error(e)) {
-                throw new Error(`Error in S3: ${key}`)
-            }
-            throw new Error('Scanning error: Scanning retry timed out')
-        }
-    }
-
     React.useEffect(() => {
         if (focusNewRate) {
             newRateNameRef.current && newRateNameRef.current.focus()
```

Comment:
**NIT**: When adding another rate cert, the supporting documents upload section is highlighted instead of the rate cert document upload section.